src/Telemetry/Telemetry.cs

```diff
@@ -96,7 +96,7 @@ private void InitializeTelemetry(string productVersion, IConfiguration config)
                 {
                     return;
                 }
-                this._logger.LogInformation($"Sending event {eventName}");
+                this._logger.LogTrace($"Sending event {eventName}");
```

Comment:
Can this still be left as LogInformation?

Reply:
Why do you think this should be left as is? Currently having these as info fills up the logs with tons of these events (especially for the trigger binding) - and customers don't generally care about us sending telemetry events. 

(I might just completely remove these later on but at least having them as Trace for right now hides them in most cases)

Reply:
I was coming from the differentiation point between trace and information. I see your point, thanks for clarifying.